[2024-02-07T00:38:53.617+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: censoEnade.cria_diretorios manual__2024-02-07T00:38:51.827131+00:00 [queued]>
[2024-02-07T00:38:53.627+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: censoEnade.cria_diretorios manual__2024-02-07T00:38:51.827131+00:00 [queued]>
[2024-02-07T00:38:53.628+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-02-07T00:38:53.649+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): cria_diretorios> on 2024-02-07 00:38:51.827131+00:00
[2024-02-07T00:38:53.655+0000] {standard_task_runner.py:60} INFO - Started process 1435 to run task
[2024-02-07T00:38:53.658+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'censoEnade', 'cria_diretorios', 'manual__2024-02-07T00:38:51.827131+00:00', '--job-id', '61', '--raw', '--subdir', 'DAGS_FOLDER/censoEnade.py', '--cfg-path', '/tmp/tmphg9wzjjo']
[2024-02-07T00:38:53.659+0000] {standard_task_runner.py:88} INFO - Job 61: Subtask cria_diretorios
[2024-02-07T00:38:53.721+0000] {task_command.py:423} INFO - Running <TaskInstance: censoEnade.cria_diretorios manual__2024-02-07T00:38:51.827131+00:00 [running]> on host 113c724a1462
[2024-02-07T00:38:53.818+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Néfi Fernandes' AIRFLOW_CTX_DAG_ID='censoEnade' AIRFLOW_CTX_TASK_ID='cria_diretorios' AIRFLOW_CTX_EXECUTION_DATE='2024-02-07T00:38:51.827131+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-02-07T00:38:51.827131+00:00'
[2024-02-07T00:38:53.821+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-02-07T00:38:53.823+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'mkdir -p /opt/*** /bronze/date=2024-02-07 /opt/*** /silver/date=2024-02-07 /opt/*** /gold/date=2024-02-07']
[2024-02-07T00:38:53.836+0000] {subprocess.py:86} INFO - Output:
[2024-02-07T00:38:53.839+0000] {subprocess.py:93} INFO - mkdir: cannot create directory ‘/bronze’: Permission denied
[2024-02-07T00:38:53.840+0000] {subprocess.py:93} INFO - mkdir: cannot create directory ‘/silver’: Permission denied
[2024-02-07T00:38:53.840+0000] {subprocess.py:93} INFO - mkdir: cannot create directory ‘/gold’: Permission denied
[2024-02-07T00:38:53.841+0000] {subprocess.py:97} INFO - Command exited with return code 1
[2024-02-07T00:38:53.853+0000] {taskinstance.py:2698} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 433, in _execute_task
    result = execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/bash.py", line 212, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2024-02-07T00:38:53.857+0000] {taskinstance.py:1138} INFO - Marking task as FAILED. dag_id=censoEnade, task_id=cria_diretorios, execution_date=20240207T003851, start_date=20240207T003853, end_date=20240207T003853
[2024-02-07T00:38:53.874+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 61 for task cria_diretorios (Bash command failed. The command returned a non-zero exit code 1.; 1435)
[2024-02-07T00:38:53.911+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-02-07T00:38:53.939+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
